src/subresource-signing.js

```diff
@@ -39,8 +54,10 @@ var srs = {
         /**
          * Assume the contents is valid, and attach the script
          * to the document so it runs.
+         *
+         * @param contents
          */
-        attach_script: function srs_attach_script(contents) {
+        attach_script: function(contents) {
```

Comment:
They're not useful since they only work inside the function in this case or as a quirk in certain versions of IE and we've already sworn them off with all this ES6 fanciness.